### ISSUE TITLE
Logged in view shows even when the login fails

### ISSUE THREAD

**Issue by elevu** [Clinical-Genomics/trailblazer#103]

**Comment by henrikstranneheim** [comment]
Is this something you can look into @elevu? What do you think of this @patrikgrenfeldt ?

**Comment by elevu** [comment]
Yes, I can! For clarity: It's just the view with the logged in navbar showing, calls to the backend are anyway failing (forbidden) so no data is shown.

**Comment by patrikgrenfeldt** [comment]
I think this is a minor since it is an internal application but if you have a clear idea how to fix it @elevu, then I think we should just fix it 🔧

**Comment by henrikstranneheim** [comment]
Agreed! Fixing it would be great!

**Comment by elevu** [comment]
The library used for al the logins flow is deprecated. It would be easier to just rewrite the app from scratch than trying to update package by package. Is it Ok if I proceed rewriting the frontend and separate it from the trailblaze backend in another repo?

**Comment by henrikstranneheim** [comment]
Sounds great to me! Please make a draft plan and let's discuss it with @jemten @eriksjolund and @patrikgrenfeldt

**Comment by Mropat** [comment]
Is this still relevant?
